src/index.tsx

```diff
@@ -31,6 +32,8 @@ export * from './layout/section/illustratedSection'
 export * from './layout/section/mediaSection'
 export * from './layout/section/slideSection'
 export * from './layout/section/textFieldsSection'
+export * from './layout/section/SEALandingLayout'
```

Comment:
The build is failing on this test, I suspect a case issue
![Screen Shot 2021-02-03 at 14 20 17](https://user-images.githubusercontent.com/373381/106752983-59bb3500-662b-11eb-999c-917b72f73251.png)

Reply:
Thanks for looking into it! I really had no idea why is it falling :S

Reply:
Still failing 🦺

Reply:
🟢